runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsConfig.java

```diff
@@ -174,7 +174,7 @@ default int outlineFeather()
 		position = 7,
 		keyName = "npcToHighlight",
 		name = "NPCs to Highlight",
-		description = "List of NPC names to highlight"
+		description = "List of NPC names to highlight<br>Separate entries with commas (,)<br>Click anywhere in plugin to reflect changes"
```

Comment:
```suggestion
		description = "List of NPC names to highlight. Format: (NPC), (NPC)"
```

This is the format used by ground items.